examples/sw/led/Makefile

```diff
@@ -0,0 +1,59 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Generate a baremetal application
+
+PROGRAM ?= led
+PROGRAM_CFLAGS = -Wall -g -Os
+ARCH = rv32imc
+# ARCH = rv32im # to disable compressed instructions
+SRCS = $(PROGRAM).c
+
+CC = /tools/riscv/bin/riscv32-unknown-elf-gcc
+
+OBJCOPY ?= $(subst gcc,objcopy,$(wordlist 1,1,$(CC)))
+OBJDUMP ?= $(subst gcc,objdump,$(wordlist 1,1,$(CC)))
+
+LINKER_SCRIPT ?= link.ld
+CRT ?= crt0.S
+CFLAGS ?= -march=$(ARCH) -mabi=ilp32 -static -mcmodel=medany \
+	-fvisibility=hidden -nostdlib -nostartfiles $(PROGRAM_CFLAGS)
+
+OBJS := ${SRCS:.c=.o} ${CRT:.S=.o}
+DEPS = $(OBJS:%.o=%.d)
+
+OUTFILES = $(PROGRAM).elf $(PROGRAM).vmem $(PROGRAM).bin $(PROGRAM).dis
+
+all: $(OUTFILES)
+
+$(PROGRAM).elf: $(OBJS) $(LINKER_SCRIPT)
+	$(CC) $(CFLAGS) -T $(LINKER_SCRIPT) $(OBJS) -o $@ $(LIBS)
+
+%.dis: %.elf
+	$(OBJDUMP) -SD $^ > $@
+
+# Note: this target requires the srecord package to be installed.
+# XXX: This could be replaced by objcopy once
+# https://sourceware.org/bugzilla/show_bug.cgi?id=19921
+# is merged.
```

Comment:
This bug is already merged now, but I'm not sure if the implementation has been cleaned up enough to be usable. And it's not really available in a standard toolchain yet, so slighly reword this to read "is merged" -> "is widely available"

Reply:
Good to know. I guess this would also simplify `link.ld` in regards to the alignment of the sections.